rllib/agents/ddpg/ddpg_policy.py

```diff
@@ -291,7 +291,7 @@ def __init__(self, observation_space, action_space, config):
             self.config,
             self.sess,
             obs_input=self.cur_observations,
-            action_sampler=self.output_actions,
+            sampled_action=self.output_actions,
```

Comment:
Try to avoid unnecessary DeveloperAPI changes, though this one looks ok.

Reply:
Wanted to underline the difference now between `sampled_actions` and `input_actions` (for loop).